openedx_lti_tool_plugin/models.py

```diff
@@ -208,31 +208,48 @@ def __str__(self) -> str:
         return f'<LtiProfile, ID: {self.id}>'
 
 
-class CourseAccessConfiguration(models.Model):
-    """Course access configuration.
+class LtiToolConfiguration(models.Model):
+    """LTI Tool Configuration.
 
-    A model to store LTI tool course access configuration.
+    A model to store LTI tool configuration.
     """
 
     EXAMPLE_ID_LIST = 'Example: ["id-1", "id-2", ...]'
 
+    class UserProvisioningMode(TextChoices):
+        """Enumeration for user provisioning modes."""
+
+        NEW_ACCOUNTS_ONLY = 'new_only', _('New accounts only (automatic)')
+        EXISTING_AND_NEW = 'existing_and_new', _('Existing and new accounts (prompt)')
+        EXISTING_ONLY = 'existing_only', _('Existing accounts only (prompt)')
+
     lti_tool = models.OneToOneField(
         LtiTool,
         on_delete=models.CASCADE,
-        related_name='openedx_lti_tool_plugin_course_permission',
+        related_name=f'{app_config.name}_lti_tool_configuration',
         verbose_name=_('LTI Tool'),
     )
     allowed_course_ids = models.TextField(
         verbose_name=_('Allowed Course IDs'),
         help_text=_(f'List of allowed courses IDs. {EXAMPLE_ID_LIST}'),
         default=[],
     )
+    user_provisioning_mode = models.CharField(
+        max_length=50,
+        choices=UserProvisioningMode.choices,
+        default=UserProvisioningMode.NEW_ACCOUNTS_ONLY,
+        verbose_name=_('User Provisioning Mode'),
+        help_text=_('Determines how user accounts are provisioned during LTI authentication.'),
```

Comment:
```suggestion
        help_text=mark_safe(_("""
        <p>Determines how user accounts are provisioned during an LTI launch:</p>
        <ul>
          <li><strong>New accounts only (automatic)</strong> - A new account will automatically be created for the user during first launch. This account will then be reused on subsequent launches.</li>
          <li><strong>Existing and new accounts (prompt)</strong> - The user will be prompted to either link an existing account, or have a new account created for them. Subsequent launches will use whichever account is the result of this choice.</li>
          <li><strong>Existing accounts only (prompt)</strong> - Only existing accounts may be used. If the user does not have an existing account to use, they will be unable to access the shared resources.</li>
        </ul>
        """))
```
@Serafin-dev What do you think about adding a description to what each option does?.

I'm not sure if mark_safe will make the HTML work, if it doesn't just keep it has a multiline gettext string, make_text can be imported from `django.utils.safestring`